### DISCUSSION TITLE
Local IDE setup for importing code

### DISCUSSION THREAD

**Discussion by pg30** [featurehub-io/featurehub#881] [Q&A] [answered]
I am trying to clone the repo in intellij IDE and building it but ran into a bunch of import problems and code navigation is not working properly. Is there a guide of how to do this setup for contributing or a list of dependencies to install. 
Thanks in advance

**Comment by rvowles** [comment, marked as answer]
Hey there!

if you follow the instructions here and make sure it all builds and installs for you? https://docs.featurehub.io/featurehub/latest/developers.html#_building_from_the_command_line

Then you should be able to load  the pom.xml into IntelliJ. That is the tool we use, please feel free to join our Slack channel or ask further questions here :-) (or both!)
